### PR TITLE
FEAT: Add known issues and limitation section

### PR DESCRIPTION
**Pull request by dipinknair**
Currently this is a template. More information will be added.

image

### CODE REVIEW

File: doc/source/kil/index.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. _ref_known_issues_and_limitation:
+
+Known issue and limitations
+===========================
+
+Here are some known issues, limitations, and things to be aware of.
```
Comment (sbradrick) [inline doc/source/kil/index.rst:6]:
```suggestion
Note the following known issues and limitations. Certain features may not be fully compatible across different versions of Mechanical and PyMechanical. 
```

File: doc/source/kil/index.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        :link: pymechanical
+        :link-type: doc
+        :text-align: center
+
+        Known issues and limitations of PyMechanical
+
+Support
+=======
+
+If you encounter any issues or limitations that hinder your work, please reach out to our support team immediately.
```
Comment (sbradrick) [inline doc/source/kil/index.rst:42]:
```suggestion
If you encounter any issues or limitations that hinder your work, please contact our support team immediately.
```

File: doc/source/kil/pymechanical.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+We suggest to use latest stable version of PyMechanical.
+Any issues and limitations for previous versions are not currently documented.
+If you find any other, please create an issue `here <pymechanical_issue_>`_
+
+v0.11.x
+^^^^^^^^
+- Remote solve (**My Computer Background**) is only available in Windows platform with embedding method
+- ``App.plot()`` visualization feature is available only for geometry. You cannot plot mesh or results
+  but you can export the plots for this data as images or an animation.
```
Comment (koubaa) [inline doc/source/kil/pymechanical.rst:17]: you can't export "the plots", you can export images and animations

File: doc/source/kil/pymechanical.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- PyMechanical does not support the use of beta features.
```
Comment (koubaa) [inline doc/source/kil/pymechanical.rst:18]: this is not true, Pymechanical can use some beta features

File: doc/source/kil/pymechanical.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Known issues and limitations
+----------------------------
+
+We suggest to use latest stable version of PyMechanical.
+Any issues and limitations for previous versions are not currently documented.
+If you find any other, please create an issue `here <pymechanical_issue_>`_
+
+v0.11.x
+^^^^^^^^
+- Remote solve (**My Computer Background**) is only available in Windows platform with embedding method
```
Comment (koubaa) [inline doc/source/kil/pymechanical.rst:15]: Didn't we fix this?

**Review by koubaa — APPROVED** [review]

File: doc/source/kil/pymechanical.rst (hunk 5 above)
Comment (dipinknair) [reply doc/source/kil/pymechanical.rst]: Yes. Removed it

**Pull request merged**